### DISCUSSION TITLE
change from div to body in getHtml() on latest version

### DISCUSSION THREAD

**Discussion by handhikadj** [GrapesJS/grapesjs#4901] [General]
I notice when I add classes or decorations in the body, it will print as div in `editor.getHtml()`
how do I change it to body tag?

```js
grapesjs.init({
  storageManager: {
    type: 'remote',
    options: {
      remote: {
        onStore: (storeData, gEditor) => {
          // this prints: <div><div>my text</div></div>
          console.log(gEditor.getHtml())
          
          // what I want: <body><div>my text</div></body>
        }
      }
    }
  }
})
```

**Comment by artf** [comment]
On the latest version I don't see any issue https://jsfiddle.net/f5nco0hj/2/

**Reply by handhikadj** [reply]
it works now somehow. not sure what caused this issue back then.
update: still getting the issue.
I need to try with the storageManager and make the sandbox of it but don't know how to make the sandbox with the API
